### ISSUE TITLE
rclone bash completion with encrypted config

### ISSUE THREAD

**Issue by braderhart** [rclone/rclone#3767]
#### What is the problem you are having with rclone?
Bash completion is no longer working for me while using an encrypted configuration file. It keeps prompting me for a password when I try to tab-complete. I think this is likely due to the newly added remote completion function. Is there a way to disable this?

#### What is your rclone version (output from `rclone version`)
rclone v1.50.2
- os/arch: linux/amd64
- go version: go1.13.4

#### Which OS you are using and how many bits (eg Windows 7, 64 bit)
Linux rad 5.3.13-arch1-1 #1 SMP PREEMPT Sun, 24 Nov 2019 10:15:50 +0000 x86_64 GNU/Linux

####  Which cloud storage system are you using? (eg Google Drive)
Google Drive

#### The command you were trying to run (eg `rclone copy /tmp remote:tmp`)
rclone sync

#### A log from the command with the `-vv` flag (eg output from `rclone -vv copy /tmp remote:tmp`)
```
$ rclone sync Enter configuration password:
password:
```

**Comment by ncw** [comment]
Ah!

A workaround would be to set the config password `export RCLONE_CONFIG_PASS=pass` however I expect that isn't what you want.

The proper solution for this would be to call rclone with `--ask-password=false` in the bash completion script to make sure it doesn't try to ask for the config password.

What do you think @sgtpep ?

**Comment by braderhart** [comment]
Thanks. You are correct. Hoping to see a way to disable the remote completion for encrypted configs in v1.51.

**Comment by sgtpep** [comment]
@ncw Yes, probably, adding `--ask-password=false`, as you've suggested, after `command rclone listremotes` in `__rclone_custom_func` would help. Unfortunately, I don't have enough time to reproduce it properly. @braderhart Would it be too much to ask you to try adding it to your bash completion script to see if it eliminates the problem?

**Comment by braderhart** [comment]
This is the error I get after making the change and trying to tab complete:

```
rclone sync --dry-run 2019/11/28 20:35:17 Failed to load config file "/home/user/.config/rclone/rclone.conf": unable to decrypt configuration and not allowed to ask for password - set RCLONE_CONFIG_PASS to your configuration password
```

**Comment by sgtpep** [comment]
@braderhart Thank you! At least it doesn't prompt for a password, right? So we only need to suppress this error message. I'd got with this change: https://github.com/rclone/rclone/pull/3773/commits/c8741f24a14443f6e65eb2ddb6b209baa6e7b1d1.

**Comment by ncw** [comment]
That change works perfectly for me so I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.51

Thank you @sgtpep for fixing this
